ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activitySection.component.ts

```diff
@@ -64,6 +64,7 @@ import { ConfigurationService } from '../../services/configuration.service';
                                           [readonly]="readonly"
                                           [activityGuid]="activityGuid"
                                           (validStatusChanged)="updateEmbeddedComponentValidationStatus(1, $event)"
+                                          [validationRequested]="validationRequested"
```

Comment:
We have been using this input param with data entry components. Will need to do something similar for validation of missing required fields.